index.html

```diff
@@ -45,7 +48,7 @@ <h2>I am a Software Developer</h2>
     </ul>
   </section>
```

Comment:
- [ ] There's an issue with this piece of code. Due to the animation that you added to your "Hey there I'm Emmanuel" text, when you go on the page and click the menu icon, the menu appears but the animation is still going on and it is being shown on the top of the menu. Once the animation ends, the text disappears from the menu as it should. Here's an image so you can understand it a bit better.
![Captura de pantalla 2022-04-12 195819](https://user-images.githubusercontent.com/59100477/163024763-ebe3d7c8-303c-4331-8cee-0b994ac23930.jpg)

Please work on this issue. You could either change the z-index of your text to be less than your menu or make the animation disappear once you click the menu button. Or if you can think of a different solution it would be also nice!